### DISCUSSION TITLE
Webui, from another pc

### DISCUSSION THREAD

**Discussion by sarmeni** [garethgeorge/backrest#447] [Q&A] [answered]
hi, I installed the raspberry os lite backrest, and I can't connect via http://ipaddress:9898, how can I set the host?
Before version 1.0.0 I could do it, now it doesn't work anymore...

**Comment by garethgeorge** [comment]
Hey, BACKREST_PORT can be set to 0.0.0.0:9898 to listen on all interfaces

**Comment by sarmeni** [comment]
Where i can set this value? I no have docker installation, but script installation...

Thanks

**Reply by garethgeorge** [reply]
What OS are you running?

**Comment by sarmeni** [comment]
Raspberry OS lite on rapsberry pi 4

**Reply by garethgeorge** [reply, marked as answer]
You'll want to edit the port specified in the systemd unit created for you, you can find that at `/etc/systemd/system/backrest.service`

**Reply by sarmeni** [reply]
Thank you, it works perfectly
